lib/tasks/production-copy.rake

```diff
@@ -22,7 +22,7 @@ namespace :production_copy do
   end
 
   desc "Get a local copy of production (no cleansing)"
-  task fetch_db: [:environment]  do
+  task fetch_db: [:environment] do
```

Comment:
You don't need an array, just use:

    task fetch_db: :environment do

and below for the `cleanse` task as well.